style.css

```diff
@@ -405,3 +448,229 @@ textarea::placeholder {
   place-items: center;
   margin-top: 55px;
 }
+
+@media screen and (min-width: 768px) {
+  .landing-page {
+    width: 1585px;
+    margin: auto;
+    height: 981px;
+    border-radius: 0 0 0 70px;
+  }
+
+  .header-nav {
+    width: 100%;
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
+    top: 0;
+  }
+
+  .logo {
+    position: relative;
+    margin-bottom: 70px;
+  }
+
+  .logo-text {
+    text-decoration: none;
+  }
+
+  .menu {
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .nav-items {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: relative;
+    bottom: 50px;
+    right: 120px;
+    gap: 15px;
+  }
+
+  .nav-link {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+    justify-content: space-between;
+  }
+
+  #hamburger {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url("./assets/images/header-bg.png");
+    background-repeat: no-repeat;
+    background-position: 20px 0;
+    height: 825px;
+    width: 1585px;
+  }
+
+  #headline-paragraph,
+  #headline-text {
+    width: 920px;
+  }
+
+  #heading-social,
+  #icons {
+    align-self: flex-start;
+    position: relative;
+    left: 308px;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .heading-social {
+    margin-left: 25px;
+    padding-left: 0;
+  }
+
+  .icons {
+    margin-left: 0;
+  }
+
+  .projects-section {
+    display: none;
+  }
+
+  .projects-section-desktop {
+    display: grid;
+    grid-template-rows: repeat(4, 1fr);
+    grid-template-columns: repeat(1fr);
+    place-items: center;
+    margin-top: 100px;
+    gap: 50px;
+    margin-left: 190px;
+  }
+
+  .projects-section-desktop .card {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid #1e315f;
+    padding: 24px;
+    height: 496px;
+    width: 1156px;
```

Comment:
- [ ] As I mentioned the reasons earlier, kindly consider removing the exact widths on your containers.